packages/ui-extensions/src/surfaces/admin/components/ProgressIndicator/ProgressIndicator.doc.ts

```diff
@@ -0,0 +1,48 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'ProgressIndicator',
+  description:
+    'Use this component to notify merchants that their action is being processed or loaded.',
+  requires: '',
+  thumbnail: 'progressindicator-thumbnail.png',
+  isVisualComponent: true,
+  type: '',
+  definitions: [
+    {
+      title: 'ProgressIndicatorProps',
+      description: '',
+      type: 'ProgressIndicatorProps',
+    },
+  ],
+  category: 'Components',
+  subCategory: 'Media',
+  defaultExample: {
+    image: 'progressindicator-default.png',
+    codeblock: {
+      title: 'Simple spinner example',
+      tabs: [
+        {
+          title: 'React',
+          code: '../../../../../../ui-extensions-react/src/surfaces/admin/components/ProgressIndicator/examples/basic-ProgressIndicator.example.tsx',
```

Comment:
Man, it would be really nice if we could alias this (not for this PR, just complaining)